### DISCUSSION TITLE
How to display charts with float values?

### DISCUSSION THREAD

**Discussion by EdaDeniz** [reaviz/reaviz#36] [Q&A]
How can I display charts with float numbers? When I use it displays as Integer. But I need float numbers. Here is my code:

```
 <RadialGauge
                data={datam}
                startAngle={0}
                endAngle={Math.PI * 2}
                height={250}
                width={550}
                minValue={0}
                maxValue={max}
                fill={"white"}
                series={
                    <RadialGaugeSeries
                        arcWidth={10}
                        colorScheme={[
                            '#00ECB1',
                            '#DF8D03',
                        ]}/>
                }
            />
```
And this is my output:

![image](https://user-images.githubusercontent.com/42175967/105499109-0c0d0700-5cd2-11eb-8b8e-1e2a8f4fcdfa.png)

**Comment by amcdnl** [comment]
It looks like I have this hardcoded here - https://github.com/reaviz/reaviz/blob/master/src/RadialGauge/RadialGaugeSeries/RadialGaugeValueLabel.tsx#L29

I think it would be best to expose this to the user as a prop on this component. Can you make a PR for that?
